features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/dbscripts/mysql.sql

```diff
@@ -1067,8 +1067,8 @@ CREATE TABLE IDN_REMOTE_FETCH_CONFIG (
 )ENGINE INNODB;
 
 CREATE TABLE IDN_REMOTE_FETCH_REVISIONS (
-	ID INT NOT NULL AUTO_INCREMENT,
-	CONFIG_ID INT NOT NULL,
+    ID VARCHAR(255) NOT NULL,
```

Comment:
Let's fix this indentation issue

Reply:
Fixed in https://github.com/wso2/carbon-identity-framework/pull/3143